pyevmasm/__main__.py

```diff
@@ -18,9 +18,26 @@ def main():
                         help='Input file, default=stdin')
     parser.add_argument('-o', '--output', nargs='?', default=sys.stdout, type=argparse.FileType('w'),
                         help='Output file, default=stdout')
+    parser.add_argument('-f', '--fork', nargs=1, default=DEFAULT_FORK, type=str,
+                        help='Fork, default: byzantium. Possible: [pre-byzantium, byzantium, constantinople].'
+                             'Also an unsigned block number is accepted to select the fork.')
 
     args = parser.parse_args(sys.argv[1:])
 
+    accepted_forks = ['pre-byzantium', 'byzantium', 'constantinople']
+    arg_fork = args.fork[0].lower()
```

Comment:
```suggestion
    arg_fork = args.fork.lower()
```

Reply:
Hi @disconnect3d  that was my original implementation ;) When you set argparse required argument parameters to 1, it becomes an array.